### ISSUE TITLE
我想引用一下你的top k算法，我该怎么弄

### ISSUE THREAD

**Issue by MeetingDream** [oval-group/smooth-topk#7]
我想引用一下你的top k算法，我该怎么弄

**Comment by lberrada** [comment]
Hi, please see #2 for a simple example to use the loss function in pytorch.
